crossref-ojs-manual/en/crossmark.md

```diff
@@ -0,0 +1,115 @@
+
+For more information information on creating a Crossref Policy, see the [Crossmark Policy Page](https://www.crossref.org/education/crossmark/crossmark-policy-page/)
+
+**Step 2: Deposit Crossmark Metadata**
+
+As a best practice and for most users, this would be done with the general deposit to Crossref for new publication and updates.
+
+Again, you will need to deposit the following: new content, updated content, Crossmark policy page.
+
+It is also encouraged that you deposit additional metadata when possible, which may include funding data, licensing information, ORCID IDs, Clinical trial numbers, and any additional custom metadata such as publication history, similarity check deposit, and much more.
+
+Some additional information may not be included in the Crossref XML export at this time. To see how other users have made code modification to export this additional information, please see the following forum posts.
+
+[Licensing Information](https://forum.pkp.sfu.ca/t/crossmark-support/1375/5)
+
+**Step 3: Add Crossref DOI to HTML pages and PDF**
+
+The metadata DOI should be added within the HTML header. This may already be added if using other metadata plugins.
+
+```html
+<meta name=”dc.identifier” content =”10.3333/1212212”>
+```
+
+Crossmark metadata also needs to be added to your PDF content. Minimally, this metadata must include the DOI of the content and the optional Crossmark domain(s). The link structure for the PDF should look like this:
+
+`https://crossmark.crossref.org/dialog/?doi=10.5555/12345678&domain=pdf&date_stamp=2017-01-14`
+
+where:
+
+- *doi* is the DOI of the content item
+- *domain* tells the Crossmark system what kind of static content the link is coming from, and will change for different static formats (such as epub)
+- *date_stamp* tells the Crossmark system the date on which a last Major Version of the PDF was generated. In most cases, this will be the date the article was published. However, when a member makes significant corrections to a PDF in-situ (no notice issued, and no new version of the work with a new DOI) then the *date_stamp* should reflect when the PDF was regenerated with the corrections. The system will then use the *date_stamp* in order to tell whether the reader needs to be alerted to updates or not. The *date_stamp* argument should be recorded in the form YYYY-MM-DD (learn more about ISO 8601).
```

Comment:
Should "(learn more about ISO 8601)" at the end of this line be a link to something?

Reply:
Probably, but this is fine. I'll get to a bigtime edit here soon.